### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by oceaneLonneux**
It is done at 90%. I just need to get the timer right, but the main work is done!

### CODE REVIEW

File: artwork/artwork.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <script
+  src="https://code.jquery.com/jquery-3.3.1.min.js"
+  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+  crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (AEJackson) [inline artwork/artwork.html]: check indentation

File: Play/play.html [hunk 2]
```diff
@@ -6,7 +6,9 @@
   src="https://code.jquery.com/jquery-3.3.1.min.js"
   integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
   crossorigin="anonymous"></script>
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (AEJackson) [inline Play/play.html]: have you included bootstrap.min.css 2 times?

File: index.html [hunk 3]
```diff
@@ -3,7 +3,12 @@
   <head>
     <meta charset="utf-8">
     <!-- bootstrap -->
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <script type="text/javascript" src="index.js">
+    </script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (AEJackson) [inline index.html]: your JS file should come after JQuery's & bootstrap's
also indent you jQuery and bootstrap script tags

**Review by AEJackson — CHANGES_REQUESTED** [review]

**Comment by oceaneLonneux** [comment]
Did the changes of indentation you requested

File: Play/play.js [hunk 4]
```diff
@@ -1,11 +1,117 @@
...
+          }, 1000)
+        }
+      }
+      clearInterval(timerId);
+    }//end of else if
+  }, 1000); //end of set interval
 
 
 
+// make an array of [0 1 2]
```
Comment (richgurney) [inline Play/play.js]: No commented out code

**Review by richgurney — CHANGES_REQUESTED** [review]

**Comment by oceaneLonneux** [comment]
Took off the commented code in play.js

File: index.html [hunk 5]
```diff
@@ -2,30 +2,25 @@
...
     <meta charset="utf-8">
-    <!-- bootstrap -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-    <script type="text/javascript" src="index.js">
-    </script>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
-  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
```
Comment (AEJackson) [inline index.html]: index.js script tag is no longer included is this an oversight ?

**Review by AEJackson — CHANGES_REQUESTED** [review]

**Comment by oceaneLonneux** [comment]
I am not using JS with my index.html, therefore why I don't have a script for that anymore.

File: artwork/artwork.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="artwork.css">
+    <title>Game - Play</title>
+  </head>
+  <body>
+    <div class="container-fluid">
+      <h1>These things happen</h1>
+    </div>
+    </script>
```
Comment (AEJackson) [inline artwork/artwork.html]: orphan </script> tag (is has no opening tag associated with it)

File: index.html [hunk 7]
```diff
@@ -2,25 +2,48 @@
...
+                      <ul>
+                        <li>You only have x seconds to play. Save your life and score!</li>
+                        <li> </li>
+                        <li> </li>
+                      </ul>
+                    </div>
+                    <div class="modal-footer">
+                      <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                    </div>
+                    </div>
```
Comment (AEJackson) [inline index.html]: check indentation

**Review by AEJackson — CHANGES_REQUESTED** [review]

**Comment by oceaneLonneux** [comment]
Did the changes requested. The game is working, I just need to work a bit more on LocalStorage but I almost understand the logic of it!

File: Play/play.js [hunk 8]
```diff
@@ -1,11 +1,125 @@
...
+  }, 800)
+
+  var timerInterval = setInterval(function(){
+    if (timeLeft != 0) {
+      // console.log(timeLeft);
+      timeLeft--;
+      $(".playerScore").html(playerScore); //show the score
+      $("score").each(function(index){
+        $(this).data("score", index)
+      }) //record the score
```
Comment (richgurney) [inline Play/play.js]: Commented out code here

File: Play/play.js [hunk 9]
```diff
@@ -1,11 +1,125 @@
...
 
-});
+  var monstInt = setInterval(function() {
+    // console.log("running again and again  ")
+    setValue();
+  }, 800)
+
+  var timerInterval = setInterval(function(){
+    if (timeLeft != 0) {
+      // console.log(timeLeft);
```
Comment (richgurney) [inline Play/play.js]: Comments Here too?

File: index.html [hunk 10]
```diff
@@ -2,25 +2,41 @@
...
+              <button type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#myModal">Instructions</button>
+              <!-- Modal -->
+              <div class="modal fade" id="myModal" role="dialog">
+                <div class="modal-dialog">
+                  <!-- Modal content-->
+                  <div class="modal-content">
+                    <div class="modal-header">Header <!-- end of modal header -->
+                      <button type="button" class="close" data-dismiss="modal">&times;</button>
+                        <h4 class="modal-title">Instructions</h4>
+                          <div class="modal-body">
```
Comment (richgurney) [inline index.html]: Can you fix the indentation here please

**Review by richgurney — CHANGES_REQUESTED** [review]

File: Play/play.html [hunk 11]
```diff
@@ -7,6 +7,7 @@
   integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
   crossorigin="anonymous"></script>
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (AEJackson) [inline Play/play.html]: you have include the bootstrap css file but not it's js file

**Comment by oceaneLonneux** [comment]
started working on the game for players 2!

File: index.html [hunk 12]
```diff
@@ -14,29 +14,40 @@
...
-                    <div class="modal-header">Header <!-- end of modal header -->
+                    <div class="modal-header">Instructions <!-- end of modal header -->
                       <button type="button" class="close" data-dismiss="modal">&times;</button>
-                        <h4 class="modal-title">Instructions</h4>
-                          <div class="modal-body">
-                          </div> <!-- end of modal body -->
-                          <div class="modal-footer">
-                            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-                          </div> <!-- end of modal footer -->
+                        <div class="modal-body">
```
Comment (AEJackson) [inline index.html]: why is this indented
check indentation.

**Review by AEJackson — CHANGES_REQUESTED** [review]

File: Play/play.js [hunk 13]
```diff
@@ -16,13 +19,28 @@ $(function (event){
   var score = JSON.parse(localStorage.getItem('playerscore'));
   var highScore = 0; //highscore
 
+  //localStorage.setItem("player1score", "player2score");
+   function displayScore() {
+     for (var i = 1; i < 2; i++) {
+     $("#test").append("<li>"+localStorage.getItem(i)+"</li>");
+     }
+   }
+     if (localStorage.getItem(1)>localStorage.getItem(2)) {
```
Comment (AEJackson) [inline Play/play.js]: check indentation

File: Play/play.js [hunk 14]
```diff
@@ -16,13 +19,28 @@ $(function (event){
...
+       $("#leaderboard").empty().text("Player 1 wins");
+       $("#leaderboard").show();
+     }else if (localStorage.getItem(1)<localStorage.getItem(2)) {
+       $("#leaderboard").empty().text("Player 2 wins");
+       $("#leaderboard").show();
+       }
```
Comment (AEJackson) [inline Play/play.js]: check indentation

File: Play/play.js [hunk 15]
```diff
@@ -16,13 +19,28 @@ $(function (event){
...
+     displayScore();
+
   function setValue() { //we're calling all the var we defined for the monsters
     console.log(playerScore); //return the score of the player on the console
     generateRandomForArray();
         $(".img img:last-child").remove() //i want the last monster to disappear
```
Comment (AEJackson) [inline Play/play.js]: check indentation

File: Play/play.js [hunk 16]
```diff
@@ -36,7 +54,6 @@ $(function (event){
 
   $(".img").click(function() {
     playerScore++; //IF you click on monster, the score rise up
-    score.html("<b  class='playerscore'> Score: " + playerScore + " </b>");
     })//end enemy click function
```
Comment (AEJackson) [inline Play/play.js]: check indentation

File: Play/play.js [hunk 17]
```diff
@@ -68,43 +85,72 @@ $(function (event){
...
-  var timerInterval = setInterval(function(){
+  var timerInterval =  setInterval(function(){
     if (timeLeft != 0) {
       timeLeft--;
-      $(".playerScore").html(playerScore); //show the score
+      var player1score = $(".player1score").html(playerScore); //show the score
+      localStorage.setItem("player1score", player1score);
       $("score").each(function(index){
         $(this).data("score", index)
+      score.html("<b  class='playerscore1'> Score: " + playerScore1 + " </b>");
```
Comment (AEJackson) [inline Play/play.js]: check indentation

File: Play/play.js [hunk 18]
```diff
@@ -68,43 +85,72 @@ $(function (event){
...
+      // $(".highScore").html(highScore); //show the score
     } else if (timeLeft == 0){
       console.log("end of game flow");
       turn ++;
-      $(".playerNames").html(secondName + " turn");
+      var player2score = $(".playerNames").html("player 2 turn");
       clearInterval(timerInterval)
       clearInterval(monstInt)
+      timeLeft = 10; //Im giving the player 10s to play
       }
```
Comment (AEJackson) [inline Play/play.js]: check indentation

File: Play/play.js [hunk 19]
```diff
@@ -68,43 +85,72 @@ $(function (event){
...
+    }, 800);
+    var timerInterval2 = setInterval(function(){
+      if (timeLeft != 0) {
+        timeLeft--;
+        localStorage.setItem("player2score", playerscore);
+        var player2score = $(".player2score").html(player2score); //show the score
+        $("score").each(function(index){
+          $(this).data("score", index)
+        score.html("<b  class='player2score'> Score: " + player2score + " </b>");
+      })
```
Comment (AEJackson) [inline Play/play.js]: check indentation

File: artwork/artwork.css [hunk 20]
```diff
@@ -0,0 +1,59 @@
...
+  width: 425px;
+  display: inline-block;
+  transform: rotate(90deg);
+  margin: 20px auto;
+  /* Set width to 100% */
+  /* height: 250px;
+  display: inline-block;
+  transform: rotate(90deg); */
+}
+/*
```
Comment (AEJackson) [inline artwork/artwork.css]: remove commented out code

**Review by AEJackson — DISMISSED** [review]

**Review by richgurney — APPROVED** [review]

**Review by richgurney — APPROVED** [review]

**Pull request merged**